AATAMS/AATAMS_acoustic/check_csv.py

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env python
```

Comment:
This is very close to perfection!( yes only very close, cause who knows what perfection is...)
Thanks for the time you spent on commenting on the script. This is a good reminder of the good programming practices i should embrace...